app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+    <item quantity="one">Máte %1$d kanál.</item>
+    <item quantity="other">Máte %1$d kanálů.</item>
+  </plurals>
+  <string name="listallchannels_copy">Zkopírovat</string>
+  <string name="listallchannels_share_all">Sdílet všechny kanály…</string>
+  <string name="listallchannels_share">Sdílet</string>
+  <string name="listallchannels_funding_tx">Transakce</string>
+  <string name="listallchannels_funding_tx_desc">Zobrazit spojovací transakci</string>
+  <string name="listallchannels_close">Zavřít</string>
+  <string name="listallchannels_share_subject">Data kanálu</string>
+  <string name="listallchannels_share_title">Sdílet data kanálu</string>
+  <string name="listallchannels_serialization_error">Nepodařilo se získat data kanálu</string>
+
+  <!-- //////////////// channel details //////////////// -->
+
+  <string name="channel_loading">Získávám data o kanálu…</string>
+  <string name="channel_error">Nepodařilo se zíkat detaily o tomto kanálu. Prosím, zkuste to znovu později.</string>
+
+  <!-- //////////////// menu //////////////// -->
+
+  <string name="menu_settings">Nastavení</string>
+  <string name="menu_send">Odeslat</string>
+  <string name="menu_receive">Přijmout</string>
+
+  <!-- //////////////// init wallet //////////////// -->
+
+  <string name="initwallet_create">Nová peněženka</string>
+  <string name="initwallet_restore">Obnovit peněženku</string>
+
+  <!-- //////////////// main //////////////////// -->
+
+  <string name="main_alert_electrum_connecting">Připojování k electrum serveru…</string>
+  <string name="main_alert_network_lost">Jste offline. Prosím, zkontrolujte vaše připojení…</string>
+  <string name="main_alert_lightning_connection_lost">Nepodařilo se připojit k Lightning síti…</string>
+  <string name="main_alert_settings_button">Nastavení</string>
+  <string name="main_swapin_incoming">příchozí +%1$s</string>
+  <string name="main_faq_button">FAQ</string>
+  <string name="main_stores_button">Obchody</string>
+
+  <!-- //////////////// notifications //////////////////// -->
+
+  <string name="inappnotif_set_up_pin">Vaše peněženka není chráněna.</string>
+  <string name="inappnotif_set_up_pin_action">Nastavit PIN</string>
+  <string name="inappnotif_mnemonics_never_seen">Děkujeme za používání Phoenix. Udělali jste zálohu vaší peněženky?</string>
+  <string name="inappnotif_mnemonics_never_seen_action">Zálohovat peněženku</string>
+  <string name="inappnotif_background_worker_cannot_run">Phoenix potřebuje běžet na pozadí čas od času. Ujistěte se, že vaše zařízení neukončuje agresivně tuto aplikaci kvůli šetření baterie.</string>
+  <string name="inappnotif_upgrade">Je dostupná nová aktualizace.</string>
+  <string name="inappnotif_upgrade_critical">Je dostupná kritická aktualizace. Peněženka nemusí fungovat správně pokud nebude aktualizována.</string>
+
+  <!-- //////////////// display seed //////////////// -->
+
+  <string name="displayseed_title">Slova k obnovení peněženky</string>
```

Comment:
Fráze?

Reply:
Wait, it's not really a phrase. More like a list of words.

Reply:
Yeah, you are right. I just have a feeling that we could use some sort of czech translation which would be easy to understand. Something like "kryptofraze". However it is really not a good idea to force some terminology onto users so "Slova" is ok even though it sounds a bit weird, the meaning is clear and understandable. "Slovní záloha", "Seznam slov" could be something to consider.

Reply:
"Seznam slov" makes more sense, thanks.